test/e2e/image_scp_test.go

```diff
@@ -13,41 +13,10 @@ import (
 )
 
 var _ = Describe("podman image scp", func() {
-	ConfPath := struct {
-		Value string
-		IsSet bool
-	}{}
-	var (
-		tempdir    string
-		podmanTest *PodmanTestIntegration
-	)
 
 	BeforeEach(func() {
-		ConfPath.Value, ConfPath.IsSet = os.LookupEnv("CONTAINERS_CONF")
-		conf, err := os.CreateTemp("", "containersconf")
-		Expect(err).ToNot(HaveOccurred())
-
-		os.Setenv("CONTAINERS_CONF", conf.Name())
-		tempdir, err = CreateTempDirInTempDir()
-		if err != nil {
-			os.Exit(1)
-		}
-		podmanTest = PodmanTestCreate(tempdir)
-		podmanTest.Setup()
-	})
-
-	AfterEach(func() {
-		podmanTest.Cleanup()
-
-		os.Remove(os.Getenv("CONTAINERS_CONF"))
-		if ConfPath.IsSet {
-			os.Setenv("CONTAINERS_CONF", ConfPath.Value)
-		} else {
-			os.Unsetenv("CONTAINERS_CONF")
-		}
-		f := CurrentSpecReport()
-		processTestResult(f)
-
+		// make sure connections are not written to real user config on host
+		os.Setenv("CONTAINERS_CONF", filepath.Join(podmanTest.TempDir, "containersconf"))
```

Comment:
Looks like you need to create this.

Reply:
Guess I deserve that for not running it locally first.